server/app.js

```diff
@@ -24,6 +25,7 @@ app.use(express.static(join(__dirname, "public")));
 
 app.use(authRoutes);
 app.use("/api/projects/", projectRoutes);
+app.use("/", imageUploadRoutes);

```

Comment:
I would bring the imageUploadRoutes into projectRoutes because it's related to that feature - make the call something in the vein of /api/projects/uploadImages?

Reply:
Modified API route to /api/projects/imageUpload